src/App.tsx

```diff
@@ -17,18 +78,37 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                query={query}
+                status={status}
+                onQueryChange={setQuery}
+                onStatusChange={setStatus}
```

Comment:
Consider adding `aria-label` attributes to the `TodoFilter` component's buttons to improve accessibility, especially since the ESLint rule for `control-has-associated-label` is disabled.